### PR TITLE
[Workspace] fix breadcrumbs for workspace

### PR DESCRIPTION
**Pull request by Hailong-am**
### Description

- [x] prepend workspace name and icon into breadcrumbs
- [x] remove `Home` from breadcrumbs
- [x] workspace overview page title change to `Overview`
- [x] Workspace settings page breadcrumbs update

## Changelog

- fix: [Workspace] prepend workspace name and icon to breadcrumbs

### Check List

- [x] All tests pass
  - [x] `yarn test:jest`
  - [x] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by Hailong-am** [comment]
link checker failure will handled by https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8309

**Review by SuZhou-Joe — DISMISSED** [review]

File: src/plugins/home/public/application/components/usecase_overview/search_use_case_app.tsx [hunk 1]
```diff
@@ -14,28 +14,32 @@ import {
...
 
   useEffect(() => {
+    const title = i18n.translate('home.searchOverview.title', { defaultMessage: 'Overview' });
+    const titleWithUseCase = i18n.translate('home.searchOverview.titleWithUseCase', {
+      defaultMessage: 'Search Overview',
+    });
     const breadcrumbs: EuiBreadcrumb[] = [
       {
-        text: currentWorkspace?.name || title,
+        text: isSearchUseCase ? title : titleWithUseCase,
```
Comment (ruanyl) [inline src/plugins/home/public/application/components/usecase_overview/search_use_case_app.tsx:38]: Maybe add a comment here for the business logic.

File: src/plugins/workspace/public/components/workspace_name/workspace_name_with_icon.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import React from 'react';
+import { getUseCase } from '../../utils';
+import { WorkspaceUseCase } from '../../types';
+
+interface Props {
+  workspace: WorkspaceAttribute;
+  availableUseCases: WorkspaceUseCase[];
+}
+
+export const WorkspaceNameWithIcon = ({ workspace, availableUseCases }: Props) => {
```
Comment (ruanyl) [inline src/plugins/workspace/public/components/workspace_name/workspace_name_with_icon.tsx:17]: How about call it `WorkspaceTitleDisplay`?

**Review by ruanyl — APPROVED** [review]

File: src/plugins/workspace/public/components/workspace_name/workspace_name_with_icon.tsx (hunk 2 above)
Comment (Hailong-am) [reply src/plugins/workspace/public/components/workspace_name/workspace_name_with_icon.tsx:17]: make sense, will update it accordingly.

File: src/plugins/home/public/application/components/usecase_overview/search_use_case_app.tsx (hunk 1 above)
Comment (Hailong-am) [reply src/plugins/home/public/application/components/usecase_overview/search_use_case_app.tsx:45]: sure, comments added.

**Review by ruanyl — APPROVED** [review]

**Review by SuZhou-Joe — APPROVED** [review]

**Pull request merged**